### PR TITLE
Add information about referenced assemblies to UI

### PR DESCRIPTION
**Pull request by zvirja**
I often need information about referenced assemblies when I review assemblies in NuGet packages. I've extended tool's UI to obtain this information and display it at the end:

image

Let me know if you have any concerns/wishes regarding the suggested functionality.

### CODE REVIEW

**Comment by zvirja** [comment]
@304NotModified Is this project still alive and do you review the PRs 😟

**Comment by 304NotModified** [comment]
Yes i'm alive, but on holiday now. Also NLog has my preference. 

(and probably i'm in another timezone).

Thanks for the PR! Will try to test it before the end of next week.

File: CodeExecutor/AppDomainWorker.cs [hunk 1]
```diff
@@ -91,6 +93,18 @@ public AssemblyMetaData GetAssemblyMetadata(string assemblyPath)
                 {
                     // if an exception occurs when loading custom attributes, just ignore
                 }
+
+                try
+                {
+                    data[AssemblyMetaData.ReferencedAssembliesKey] = assembly.GetReferencedAssemblies()
+                        .OrderBy(assName => assName.Name)
+                        .Aggregate(new StringBuilder(), (output, assName) => output.AppendLine(assName.FullName))
```
Comment (304NotModified) [inline CodeExecutor/AppDomainWorker.cs]: looks good, but what about `string.Join`? That's a bit more clear IMO.

**Comment by 304NotModified** [comment]
Approved, but prefer `string.join` over `Aggregate`

**Comment by 304NotModified** [comment]
> @304NotModified Is this project still alive and do you review the PRs

If you have time over, it could use some help in testing the other PRs. ;)

**Comment by zvirja** [comment]
@304NotModified I've followed up and replaced LINQ with `string.Join()` - indeed looks nicer 😉 

What should I do to get this PR merged? Are we waiting for somebody else's approval?

> If you have time over, it could use some help in testing the other PRs. ;)

I'll look over them, but am not sure I have enough tool knowledge to test them..

File: CodeExecutor/AppDomainWorker.cs (hunk 1 above)
Comment (zvirja) [reply CodeExecutor/AppDomainWorker.cs]: Solved

**Comment by 304NotModified** [comment]
2 things for approval: code review and test. I reviewed your code, so only a test (of course both not from the code creator. )

The test should test the feature/bug fix and the common actions (open, create, save package)

The other cases aren't difficult to test IMO.

**Comment by zvirja** [comment]
Meaning that now we should wait till you find some time to test the stuff? :) 

In this case it's fine - everything has been done from my side and now it's time to have a rest and simply wait :smiley:

**Comment by 304NotModified** [comment]
> Meaning that now we should wait till you find some time to test the stuff? :)

yes indeed :)

**Comment by zvirja** [comment]
@304NotModified I know that you're probably busy these days, but is there any chance you merge this PR is the nearby future? :cry: PR is really tiny, but I demand that feature very often..

**Comment by 304NotModified** [comment]
Yes, Will looks at this now!

**Comment by 304NotModified** [comment]
have to install VS 15.4 first I see... Working on that

**Comment by 304NotModified** [comment]
...no free space enough..

**Comment by zvirja** [comment]
@304NotModified Any luck to install VS? :blush:

**Pull request merged**